docs/metadata.md

```diff
@@ -0,0 +1,582 @@
+# Cachito request metadata
+
+One of Cachito's secondary (but no less important) responsibilities is reporting all
+the sources that were provided for a specific request. Cachito provides two different
+metadata formats for this purpose.
+
+The first format is the JSON response at `/api/v1/requests/{id}`, which contains all
+the information about a request. In the rest of the document, this will be referred to
+as [request JSON](#request-json).
+
+The second format is the [Content Manifest](#content-manifest) at
+`/api/v1/requests/{id}/content-manifest`. This is a JSON document as well, but contains
+only information about the packages and dependencies present in the request bundle.
+
+All the sources are also present in the request bundle downloaded from
+`/api/v1/requests/{id}/download`. The packages from the requested git repository are
+under `app/`. All the dependencies downloaded by Cachito are under `deps/`, unless
+vendoring was used, in which case they will be in some vendor location specific to the
+given package manager.
```

Comment:
In the `app/` directory in this case, right?

How about something like this:
> All the dependencies downloaded by Cachito are usually under `deps/`. However, when vendoring is used, the source code for the dependencies will be in the vendor location specific to the given package manager under the `app/` directory. For example, vendored `gomod` dependencies will be under `app/vendor/`, not `deps/`. Vendoring is optional and not all package managers support it.

Reply:
The sentences were getting a bit long, so I reworded it a little. The general idea should be captured there